src/index.ts

```diff
@@ -348,7 +354,7 @@ export default class MachinomyServerPlugin extends MiniAccountsPlugin {
 
         // isFunding should be toggled after all non-funding related aysnc operations have completed, but before any balance changes occur
         if (account.isFunding) {
-          throw new Trace(`Failed to pay account ${account.accountName}: another funding event was occuring simultaneously.`)
+          throw new Trace(`another funding event was occuring simultaneously.`)
```

Comment:
I don't think an in-flight funding transaction should stop packets from being processed. Unless funding has a different meaning here than in the XRP context (adding funds to the channel) it should happen concurrently with traffic on the plugin. An in flight fund should just prevent more fund transactions from being issued until it's done.